cirq/ops/gate_operation_test.py

```diff
@@ -73,6 +73,22 @@ def p(*q):
     eq.add_equality_group(p(b0, a1, a0, b1, c0))
 
 
+def test_gate_operation_approx_eq():
+    a = [cirq.NamedQubit('r1')]
+    b = [cirq.NamedQubit('r2')]
+
+    assert cirq.approx_eq(cirq.GateOperation(cirq.XPowGate(), a),
+                          cirq.GateOperation(cirq.XPowGate(), a))
+    assert not cirq.approx_eq(cirq.GateOperation(cirq.XPowGate(), a),
+                              cirq.GateOperation(cirq.XPowGate(), b))
+
+    assert cirq.approx_eq(cirq.GateOperation(cirq.XPowGate(exponent=0), a),
```

Comment:
Use `cirq.X(a)**t` form, since it is much shorter.

Reply:
Done. I wanted to be consistent with the rest of a file.